### ISSUE TITLE
Log4j vul (CVE-2021-44228) patches required?

### ISSUE THREAD

**Issue by nodesocket** [logdna/logdna-agent-v2#278]
Is there anything we as users of the LogDNA Helm chart and Docker containers need to patch related to the Log4j vulnerability?

Running `Image:      logdna/logdna-agent:2.2.4`.

**Comment by dhable** [comment]
tl;dr - No

LogDNA has become aware of a vulnerability with log4j, a logging utility from the Apache Logging Services project.  As part of ongoing security due diligence, LogDNA regularly scans to validate that the versions of code we have running in production are consistently kept up to date.  Nonetheless, the team has evaluated our platform and clients in the last two days and determined we are not exposed to risk from log4j at this time.  We will continue to evaluate the situation and update any related components to eliminate them from the platform entirely should they contain problematic code.  We will follow up with additional communication if any additional findings or remediations are undertaken by LogDNA.

**Comment by nodesocket** [comment]
@dhable great thanks for the prompt reply.
